frappe/auth.py

```diff
@@ -340,6 +340,11 @@ def init_cookies(self):
 	def set_cookie(self, key, value, expires=None, secure=False, httponly=False, samesite="Lax"):
 		if not secure and hasattr(frappe.local, 'request'):
 			secure = frappe.local.request.scheme == "https"
+
+		# Cordova does not work with Lax
+		if frappe.local.session.data.device == "mobile":
+			samesite = None
```

Comment:
Shouldn't this be a string?

Reply:
My mistake, that cannot happen yet. I've created [an issue](https://github.com/frappe/frappe/issues/14940) instead.